src/pages/office/cat-cpc/office.cat-cpc.page.html

```diff
@@ -0,0 +1,121 @@
+                         [activityCodeOptions]="activityCodeOptions"
+                         (activityCodeChange)="activityCodeChanged($event)" [disabled]="true">
+          </activity-code>
+
+          <!--          <div no-padding [hidden]="!(pageState.isTestOutcomeSet$ | async)">-->
+          <div no-padding>
+
+            <candidate-description [display]="pageState.displayCandidateDescription$ | async" [formGroup]="form"
+                                   [candidateDescription]="pageState.candidateDescription$ | async"
+                                   [outcome]="pageState.testOutcome$ | async"
+                                   (candidateDescriptionChange)="candidateDescriptionChanged($event)">
+            </candidate-description>
+
+            <identification [display]="pageState.displayIdentification$ | async" [formGroup]="form"
+                            [identification]="pageState.identification$ | async"
+                            [outcome]="pageState.testOutcome$ | async"
+                            (identificationChange)="identificationChanged($event)">
+            </identification>
+
+            <combination [combination]="pageState.combination$ | async"
+                         [combinationAdditionalText]="getCombinationAdditionalText(combinationAdditionalText)">
+            </combination>
+
+            <additional-information [display]="pageState.displayAdditionalInformation$ | async"
+                                    [additionalInformation]="pageState.additionalInformation$ | async"
+                                    [formGroup]="form"
+                                    [outcome]="pageState.testOutcome$ | async"
+                                    (additionalInformationChange)="additionalInformationChanged($event)">
+            </additional-information>
+
+            <assessment-report *ngIf="displayIfFail(outcome)"
+                               [assessmentReport]="pageState.assessmentReport$ | async"
+                               [formGroup]="form"
+                               [outcome]="pageState.testOutcome$ | async"
+                               (assessmentReportChange)="assessmentReportChanged($event)">
+            </assessment-report>
+
+            <cpc-debrief-card *ngIf="displayIfFail(outcome)"
+                              [overallScore]="pageState.overallScore$ | async"
+                              [question1]="pageState.question1$ | async"
+                              [question2]="pageState.question2$ | async"
+                              [question3]="pageState.question3$ | async"
+                              [question4]="pageState.question4$ | async"
+                              [question5]="pageState.question5$ | async"
+                              [testOutcome]="outcome">
+            </cpc-debrief-card>
+
+          </div>
+        </ion-grid>
+      </ion-card-content>
+    </ion-card>
+
+    <div no-padding [hidden]="!(pageState.isTestOutcomeSet$ | async)">
```

Comment:
is this needed?

Reply:
based it off another office page, but no its not required.  removed.